.github/workflows/trigger-circle-ci-workflow.yml

```
@@ -77,7 +77,8 @@ jobs:
   get-parameters:
     if: github.repository_owner == 'storybookjs'
     runs-on: ubuntu-latest
-    permissions: {}
+    permissions:
+      members: read
```

actionlint (1.7.12)

[error] unknown permission scope "members". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)